src/databricks/labs/ucx/source_code/known.json

```diff
@@ -3187,6 +3187,48 @@
     "dockerpycreds.utils": [],
     "dockerpycreds.version": []
   },
+  "duckdb": {
+    "adbc_driver_duckdb": [],
+    "adbc_driver_duckdb.dbapi": [],
+    "duckdb": [],
+    "duckdb.bytes_io_wrapper": [],
+    "duckdb.experimental": [],
+    "duckdb.experimental.spark": [],
+    "duckdb.experimental.spark._globals": [],
+    "duckdb.experimental.spark._typing": [],
+    "duckdb.experimental.spark.conf": [],
+    "duckdb.experimental.spark.context": [],
+    "duckdb.experimental.spark.errors": [],
+    "duckdb.experimental.spark.errors.error_classes": [],
+    "duckdb.experimental.spark.errors.exceptions": [],
+    "duckdb.experimental.spark.errors.exceptions.base": [],
+    "duckdb.experimental.spark.errors.utils": [],
+    "duckdb.experimental.spark.exception": [],
+    "duckdb.experimental.spark.sql": [],
+    "duckdb.experimental.spark.sql._typing": [],
+    "duckdb.experimental.spark.sql.catalog": [],
+    "duckdb.experimental.spark.sql.column": [],
+    "duckdb.experimental.spark.sql.conf": [],
+    "duckdb.experimental.spark.sql.dataframe": [],
+    "duckdb.experimental.spark.sql.functions": [],
+    "duckdb.experimental.spark.sql.group": [],
+    "duckdb.experimental.spark.sql.readwriter": [],
+    "duckdb.experimental.spark.sql.session": [
```

Comment:
Is a false positive, there is no databricks runtime in duckdb. (https://github.com/duckdb/duckdb/blob/33b44ab04fd25034fd89fb33b52261d286704b01/tools/pythonpkg/duckdb/experimental/spark/sql/session.py)

Reply:
I don't thins it's a false positive. If DuckDB is running within a DBR, then calling DuckDB on parquet storage managed by DBR will fail.

Reply:
Could you link the line that this message is pointing to? It is missing in the `known.json`

Reply:
It's probably this code:
https://github.com/duckdb/duckdb/blob/33b44ab04fd25034fd89fb33b52261d286704b01/tools/pythonpkg/duckdb/experimental/spark/sql/session.py#L217

Reply:
That is creating a table using the `DuckDBPyConnection`. Still not expect that to be influence by the Databricks runtime. I expect none of the duckdb modules to result a lint message